frontend/src/pages/spells/[slug]/index.js

```diff
@@ -0,0 +1,50 @@
+import SpellDetails from '../../../components/pages/spells/[slug]/SpellDetails'
+import { getSpells, getSpellsBySlug } from '../../../lib/load_spells';
+
+
+const Spells = ({ data, links }) => {
+  const { attributes } = data;
+
+  return (
+    <SpellDetails attributes={attributes} apiLink={links?.self} />
```

Comment:
Maybe you can also add a SpellMeta such as we have it for the [Characters](https://github.com/danielschuster-muc/potter-db/blob/master/frontend/src/pages/characters/%5Bslug%5D/index.js#L14).